pkg/controller/perconaservermongodbbackup/backup.go

```diff
@@ -91,12 +91,12 @@ func (b *Backup) Start(ctx context.Context, k8sclient client.Client, cluster *ap
 	case api.BackupStorageS3:
 		status.S3 = &stg.S3
 		if stg.S3.Prefix != "" {
-			status.Destination = stg.S3.Prefix + "/"
+			status.Destination = "s3://" + stg.S3.Bucket + "/" + stg.S3.Prefix + "/"
 		}
 	case api.BackupStorageAzure:
 		status.Azure = &stg.Azure
 		if stg.Azure.Prefix != "" {
-			status.Destination = stg.Azure.Prefix + "/"
+			status.Destination = "azure://" + stg.Azure.Container + "/" + stg.Azure.Prefix + "/"
```

Comment:
I don't like this change. `s3://operator-testing` is valid, since `s3://` is a protocol but there's no `azure://`. Azure uses `https://`. I get it, it looks nice esthetically but it's not usable. wdyt @hors ?

Reply:
@egegunes we have azure:// in PXC operator and we have the same prefix in PXB docs. xbcloud can work with `azure:// `. I want to have some consistency between our operators.